agent/tests/test_user_instrument.c

```diff
@@ -145,6 +145,28 @@ static void test_hashmap_wraprec() {
 }
 #endif /* PHP >= 7.4 */
 
+static void test_add_custom_tracer_named() {
+  nruserfn_t* wr = NULL;
+  tlib_php_request_start();
+
+  wr = nr_php_add_custom_tracer_named(NULL, 10);
+  tlib_pass_if_null("add_custom_tracer_named with NULL name", wr);
+
+  wr = nr_php_add_custom_tracer_named("function_name", 0);
+  tlib_pass_if_null("add_custom_tracer_named with length == 0", wr);
+
+  wr = nr_php_add_custom_tracer_named("function_name", -1);
```

Comment:
Im curious what this is doing - the function called accepts `size_t` which I believe is unsigned, but -1 is being passed in - does the compiler flag this?  I would think the receiving function would receive a very large positive integer unless something is happening under the hood.